app/frontend/src/__mocks__/service.ts

```diff
@@ -0,0 +1,42 @@
+import { service as originalService } from "../service";
+
+export const mockedService = {} as typeof originalService;
+
+export const service = new Proxy(
+  {},
+  {
+    get(target: {}, name: PropertyKey): any {
+      return new Proxy(
+        {},
+        {
+          get(target: {}, serviceName: PropertyKey): any {
+
+            const serviceMethod =
+              (mockedService as any)[name] &&
+              (mockedService as any)[name][serviceName];
```

Comment:
Having trouble getting my head around this. Is `name` api, conversations, requests, etc. and `serviceMethod` is getUser, listGroupChats, etc.? In that case, maybe name should be serviceName, serviceName should be methodName, and serviceMethod should be serviceMethodExists.

Reply:
Ok, except for the last one. In serviceMethod really might be a function which then will be called.